app/models/repo.js

```diff
@@ -130,18 +130,47 @@ Repo.reopen({
     return array;
   }.property(),
 
+  crons: function() {
+    var array, builds, id;
+    id = this.get('id');
+    builds = this.store.filter('build', {
+      event_type: 'cron',
+      repository_id: id
+    }, function(b) {
+      return b.get('repo.id') + '' === id + '' && b.get('eventType') === 'cron';
+    });
+    array = ExpandableRecordArray.create({
+      type: 'build',
+      content: Ember.A([])
+    });
+    array.load(builds);
+    id = this.get('id');
+    array.observe(builds);
+    return array;
+  }.property(),
```

Comment:
I’m pausing here because I need to understand better what is happening here and it’s the end of the day for me! I’ll return.